tests/test_class.cpp

```diff
@@ -510,8 +523,9 @@ CHECK_HOLDER(6, shared); CHECK_HOLDER(7, shared); CHECK_HOLDER(8, shared);
 // failures occurs).
 
 // We have to actually look into the type: the typedef alone isn't enough to instantiate the type:
-#define CHECK_BROKEN(N) static_assert(std::is_same<typename Breaks##N::type, BreaksBase<-N>>::value, \
-        "Breaks1 has wrong type!");
+#define CHECK_BROKEN(N)                                                                           \
```

Comment:
For my own benefit, merging this into the smart_holder branch, if you could minimize the whitespace changes here that would definitely be helpful. But optional.